src/config/knexfile.ts

```diff
@@ -0,0 +1,30 @@
+import { database } from './config'
+
+export const config = {
+  development: {
+    client: 'pg',
+    connection: {
+      host: 'localhost',
```

Comment:
the configuration ended up being duplicated. we have those entries in both places. what if we load host, port, user, password and database in the config (from env) file and just import it there?